src/node.js

```diff
@@ -340,7 +340,8 @@
     function _tickCallback() {
       var callback, threw, tock;
 
-      scheduleMicrotasks();
+      if (microtasksScheduled)
```

Comment:
Yeah. I thought V8 would have been a little better at inlining the conditional of the function call, but it does seem to make enough difference to require placing the conditional here.